### ISSUE TITLE
Key presses not recognized

### ISSUE THREAD

**Issue by asishm** [jupyterlab/jupyterlab#7902]
1. Click inside a cell to edit the contents
2. Press Esc to go to out of the cell
3. Esc key press is not recognized. Cannot use any cell commands like ctrl + enter or Shift + enter or esc.
4. Manually clicking outside the cell and then using Shift+enter works. However up/down arrows aren't recognized to move up/down between cells.

## Context

- Operating System and version: Windows 10
- Browser and version: Chrome 80.x
- JupyterLab version: 1.1.4

<details><summary>Browser Output</summary>
<pre>
index.js:42 Starting application in workspace: "/lab/workspaces/auto-2"
router.js:128 Routing /lab/workspaces/auto-2?clone was short-circuited by apputils:load-statedb
2default.js:56 Starting WebSocket: ws://localhost:8888/api/kernels/37dfc9b3-b094-4884-a8d9-2dca79429d2c
2default.js:56 Starting WebSocket: ws://localhost:8888/api/kernels/22085ea7-528a-4fef-8380-4d52328b8043
2default.js:1496 Kernel: connected (37dfc9b3-b094-4884-a8d9-2dca79429d2c)
2default.js:1496 Kernel: connected (22085ea7-528a-4fef-8380-4d52328b8043)
4index.js:504 Cannot execute key binding 'Enter': command 'notebook:enter-edit-mode' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
8index.js:504 Cannot execute key binding 'Shift Enter': command 'notebook:run-cell-and-select-next' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
index.js:504 Cannot execute key binding 'C': command 'notebook:copy-cell' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
index.js:504 Cannot execute key binding 'Z': command 'notebook:undo-cell-action' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
7index.js:504 Cannot execute key binding 'Enter': command 'notebook:enter-edit-mode' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
13index.js:504 Cannot execute key binding 'ArrowDown': command 'notebook:move-cursor-down' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
push.5TpB.CommandRegistry.processKeydownEvent @ index.js:446
push.2R+v.Application.evtKeydown @ index.js:340
push.2R+v.Application.handleEvent @ index.js:296
17index.js:504 Cannot execute key binding 'Shift Enter': command 'notebook:run-cell-and-select-next' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
push.5TpB.CommandRegistry.processKeydownEvent @ index.js:446
push.2R+v.Application.evtKeydown @ index.js:340
push.2R+v.Application.handleEvent @ index.js:296
15index.js:504 Cannot execute key binding 'Escape': command 'notebook:enter-command-mode' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
push.5TpB.CommandRegistry.processKeydownEvent @ index.js:446
push.2R+v.Application.evtKeydown @ index.js:340
push.2R+v.Application.handleEvent @ index.js:296
2index.js:504 Cannot execute key binding 'Enter': command 'notebook:enter-edit-mode' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
push.5TpB.CommandRegistry.processKeydownEvent @ index.js:446
push.2R+v.Application.evtKeydown @ index.js:340
push.2R+v.Application.handleEvent @ index.js:296
index.js:504 Cannot execute key binding 'ArrowDown': command 'notebook:move-cursor-down' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
push.5TpB.CommandRegistry.processKeydownEvent @ index.js:446
push.2R+v.Application.evtKeydown @ index.js:340
push.2R+v.Application.handleEvent @ index.js:296
index.js:504 Cannot execute key binding 'ArrowUp': command 'notebook:move-cursor-up' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
push.5TpB.CommandRegistry.processKeydownEvent @ index.js:446
push.2R+v.Application.evtKeydown @ index.js:340
push.2R+v.Application.handleEvent @ index.js:296
index.js:504 Cannot execute key binding 'ArrowDown': command 'notebook:move-cursor-down' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
push.5TpB.CommandRegistry.processKeydownEvent @ index.js:446
push.2R+v.Application.evtKeydown @ index.js:340
push.2R+v.Application.handleEvent @ index.js:296
index.js:504 Cannot execute key binding 'ArrowUp': command 'notebook:move-cursor-up' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
push.5TpB.CommandRegistry.processKeydownEvent @ index.js:446
push.2R+v.Application.evtKeydown @ index.js:340
push.2R+v.Application.handleEvent @ index.js:296
index.js:504 Cannot execute key binding 'ArrowDown': command 'notebook:move-cursor-down' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
push.5TpB.CommandRegistry.processKeydownEvent @ index.js:446
push.2R+v.Application.evtKeydown @ index.js:340
push.2R+v.Application.handleEvent @ index.js:296
2index.js:504 Cannot execute key binding '2': command 'notebook:change-cell-to-heading-2' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
push.5TpB.CommandRegistry.processKeydownEvent @ index.js:446
push.2R+v.Application.evtKeydown @ index.js:340
push.2R+v.Application.handleEvent @ index.js:296
index.js:504 Cannot execute key binding 'Shift Enter': command 'notebook:run-cell-and-select-next' is not enabled.
push.5TpB.CommandRegistry._executeKeyBinding @ index.js:504
push.5TpB.CommandRegistry.processKeydownEvent @ index.js:446
push.2R+v.Application.evtKeydown @ index.js:340
push.2R+v.Application.handleEvent @ index.js:296
14index.js:504 Cannot execute key binding 'Ctrl Enter': command 'notebook:run-cell' is not enabled.
</pre>
</details>

**Comment by vidartf** [comment]
> JupyterLab version: 1.1.4

Thanks for the report, and for including the log output! I cannot reproduce this behavior on lab 2.0 (try e.g. https://mybinder.org/v2/gh/jupyterlab/jupyterlab-demo/master?urlpath=lab/tree/demo). Is this something that happened once, or something you are reliably able to reproduce?

**Comment by ian-r-rose** [comment]
I have also seen behavior like this, but intermittently, and have not been able to figure out what triggers it. I'll take a closer look if I see it again.

**Comment by meeseeksmachine** [comment]
This issue has been mentioned on **Jupyter Community Forum**. There might be relevant details there:

https://discourse.jupyter.org/t/having-issues-running-cells/3563/4

**Comment by vidartf** [comment]
@ian-r-rose Well, it does say "command not enabled", so it might be related to the focus tracker?

**Comment by ian-r-rose** [comment]
Yes, that or perhaps a bug in applying/removing the `jp-mod-editMode` selector.
